CMakeLists.txt

```diff
@@ -693,13 +709,18 @@ endif()
 
 # SIRIUS
 if(CP2K_USE_SIRIUS)
+  find_package(pugixml REQUIRED)
```

Comment:
I'm not sure which problem this is trying to solve, but it is possible this is related to https://github.com/electronic-structure/SIRIUS/pull/1050?

Reply:
I will remove it. But can we merge this PR. Waiting for spack to merge all open PR can take some time and spack is not yet a requirement. Moreover, we need yet another pr for sirius.

Reply:
> Moreover, we need yet another pr for sirius.

I have an incoming PR enabling SIRIUS for the `psmp` build without `pugixml`. This dependency can then be re-enabled once we have a new version of SIRIUS (soon).

Reply:
fixed in the toolchain with a patch and mainstream with a PR. We can wait for the new release and I can remove the patch entirely.